### DISCUSSION TITLE
Use American English for coding style?

### DISCUSSION THREAD

**Discussion by ctwhome** [research-software-directory/RSD-as-a-service#204] [Q&A]
There are a bunch of times where both Organi**z**ations vs Organi**s**ations are written indistinctively.
I suggest sticking to only one style, my preference would be American English. in fact we are already using it everywhere else, i.e. color, gray, center. 

Particularly for Organi**s**ations, I believe that it comes from the legacy RSD. We might need to search and replace all the places before it gets bigger.

**Comment by ctwhome** [comment]
@ewan-escience @dmijatovic @jmaassen what is your opinion about it?

**Reply by ewan-escience** [reply]
I prefer British English over American English, but has mostly to do with preferences for the actual countries. Consistency is good here and if the majority prefers American English, I will accept that.

**Reply by dmijatovic** [reply]
Looking at the code base: database, tablenames, fieldnames then on the frontend properties in interfaces and types I prefer British English, just because all code I just mentioned uses organi**s**ations. I am ok if we decide to change it but I am concerned about the amount of work required. In addition I assumed that this decision was taken with legacy RSD and has some motivation for it.

**Reply by cmeessen** [reply]
Maybe I can add to this discussion by giving you an outside perspective: I generally also prefer British English. However, looking at the title of your institute "eScience center", from my point of view it would make more sense to use American English to be consistent.
